static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.ts

```diff
@@ -18,10 +18,7 @@ const shouldRenderLabel = (adSlotNode: HTMLElement): boolean =>
 		adSlotNode.classList.contains('ad-slot--collapse') ||
 		adSlotNode.getAttribute('data-label') === 'false' ||
 		// Don't render an ad slot label if there's one already present in the slot
-		// It's fine for a hidden toggled label to be present
-		adSlotNode.querySelectorAll(
-			'.ad-slot__label:not(.ad-slot__label--toggle.hidden)',
-		).length
+		adSlotNode.querySelectorAll('.ad-slot__label').length
```

Comment:
I would guess we no longer need this line, but there's some complexity with top-above-nav that I'm not fully across yet.

Reply:
If this works fine and with a fabric ad (`adtest=fabric_demo`) (which has it's own label inside the ad), it should be fine.

Reply:
I don't think we even need this replacement line on line 21 as we should never have an ad-slot__label div -  I think we can just delete this and amend the tests accordingly.